src/custom/controller.go

```diff
@@ -0,0 +1,175 @@
+package custom
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	coreV1 "k8s.io/api/core/v1"
+	metaV1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	appsInformers "k8s.io/client-go/informers/apps/v1"
+	"k8s.io/client-go/kubernetes"
+	appsListers "k8s.io/client-go/listers/apps/v1"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/client-go/util/workqueue"
+)
+
+const SecretMounter string = "secret-mounter"
+const DefaultContainerSecretPath string = "/etc/" + SecretMounter + "-data/"
+
+// Mandatory field
+const DeploymentLabelSecretName string = "secret-name"
+// Optional field
+const DeploymentLabelSecretKeys string = "secret-keys"
+// Secret Keys separator
+const SecretKeysSeparator string = "."
```

Comment:
consts can be grouped 
```suggestion
const (
    SecretMounter string = "secret-mounter"
    DefaultContainerSecretPath string = "/etc/" + SecretMounter + "-data/"

    // Mandatory field
    DeploymentLabelSecretName string = "secret-name"
    // Optional field
    DeploymentLabelSecretKeys string = "secret-keys"
    // Secret Keys separator
    SecretKeysSeparator string = "."
)
```